1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,7 +16,8 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
+
+let thingsICanBuy = store.filter( ( itemPrice )=> itemPrice.price < money) .map( (items)=> (items.item))
```

Comment:
nice use of chaining! You don't need the space between the `filter` and `map` though (e.g. `store.filter().map()` instead of `store.filter() .map()`